packages/golden_toolkit/lib/src/golden_builder.dart

```diff
@@ -99,6 +99,23 @@ class GoldenBuilder {
     ));
   }
 
+  ///  [addScenarioContextWithContext] will add a test with BuildContext GoldenBuilder
+  /// use as:
+  /// ..addScenarioContextWithContext(
+  ///   'Test with context',
+  ///   (context) {
+  ///     return Container();
```

Comment:
I think this example should actually use context... similar to the feedback below in the test... maybe access Theme or some other property via context?